rpcd/playbooks/roles/rpc_support/tasks/holland_preinstall.yml

```diff
@@ -24,6 +31,9 @@
 - name: Install pip dependencies
   pip:
     name: "{{ item }}"
+    state: latest
+    virtualenv: "{{ holland_venv_enabled | bool | ternary(holland_venv_bin|dirname, omit) }}"
```

Comment:
I think ternary(holland_venv_bin|dirname, omit) needs to be:
ternary(holland_venv|direname, omit). In other words, We don't want the /bin, but the virutal environment directory(it's parent directory)